plugins/identity/app/services/service_layer/identity_service.rb

```diff
@@ -9,6 +9,7 @@ class IdentityService < Core::ServiceLayer::Service
     include IdentityServices::Group
     include IdentityServices::Role
     include IdentityServices::RoleAssignment
+    include IdentityServices::AWSCredential
```

Comment:
Please leave the identity service name as before `Credential`. If we need in the future to add more credentials this module would not fit anymore. Changing the method name is enough. Thx!

Reply:
done.